pkg/rpcclient/invoker/invoker.go

```diff
@@ -192,6 +199,18 @@ func (v *Invoker) Run(script []byte) (*result.Invoke, error) {
 	return v.client.InvokeScript(script, v.signers)
 }
 
+// RunContained executes the script of the given transaction using provided
+// transaction as a script container and provided header to mock persisting
+// block of execution environment. See the documentation of
+// [RPCInvoke.InvokeContainedScript] for more details. If provided transaction
+// doesn't have signers then Invoker-specific list of signers will be used.
+func (v *Invoker) RunContained(tx *transaction.Transaction, header *block.Header, t *trigger.Type) (*result.Invoke, error) {
```

Comment:
I don't think it's useful for `Invoker`. This makes `signers` irrelevant and it's a major thing `Invoker` provides conceptually. Then the interface is about the same as the original one, it's not simplifying anything.

Reply:
> This makes signers irrelevant

Why? The common use-case to me is to bind invoker-specific signers to this request so that user don't need to specify them manually.

Reply:
Fixed.